src/index.ts

```diff
@@ -27,6 +30,18 @@ function managerEntries(entry: string[] = []) {
   return [...entry, require.resolve("./manager.mjs")];
 }
 
+const getConfigSuggestions = async (config: Configuration) => {
+  const suggestions: Partial<Configuration> = {};
+
+  const { repositoryRootDir } = await getGitInfo();
+  const storybookBaseDir = repositoryRootDir && relative(repositoryRootDir, process.cwd());
+  if (storybookBaseDir && storybookBaseDir !== config.storybookBaseDir) {
+    suggestions.storybookBaseDir = storybookBaseDir;
```

Comment:
May as well put `storybookConfigDir` in as well? It's just `configDir` below..

Reply:
The point of the ticket we're working on is just to get `storybookBaseDir` sorted before we move on to others. Just that we can detect something doesn't mean we should suggest it. `storybookConfigDir` is only useful in conjunction with `storybookBuildDir` (i.e. a prebuilt Storybook) which doesn't apply to VTA builds.